deploy.sh

```diff
@@ -40,34 +41,40 @@ npm install
 grunt build
 
 # Stop Mes Aides
-killall --user `whoami` node || echo 'No server was running'
+forever stop mes-aides || echo 'No server was running'
 
-mongo localhost --eval "db.copyDatabase('mes-aides-master', '`whoami`')"
+mongo localhost --eval "db.copyDatabase('mes-aides-master', '$USER')"
 
 # Start Mes Aides
-OPENFISCA_URL="http://localhost:$OPENFISCA_PORT" SESSION_SECRET=foobar NODE_ENV=production MES_AIDES_ROOT_URL="$PROTOCOL://$PUBLIC_HOST" PORT=$PORT MONGODB_URL="mongodb://localhost/$(whoami)" nohup node server.js >> ../mes-aides.log 2>> ../mes-aides_error.log &
+OPENFISCA_URL="http://localhost:$OPENFISCA_PORT" SESSION_SECRET=foobar NODE_ENV=production MES_AIDES_ROOT_URL="$PROTOCOL://$PUBLIC_HOST" PORT=$PORT MONGODB_URL="mongodb://localhost/$USER" forever --uid mes-aides -l ../mes-aides.log -e ../mes-aides_error.log --append start server.js
 
 cd ..
 
 # Install OpenFisca
 if ! cd openfisca
 then
-    git clone https://github.com/sgmap/openfisca.git --branch $TARGET_BRANCH
+    git clone https://github.com/sgmap/openfisca.git
     cd openfisca
 fi
 
-./update.sh $TARGET_BRANCH
+./update.sh $TARGET_BRANCH || {
+        echo "No branch $TARGET_BRANCH was found on sgmap/openfisca. Staying on current branch."
+        ./update.sh
+    }
 
 # Stop OpenFisca
-killall --user `whoami` /usr/bin/python || echo 'No OpenFisca server was running'
-# Start OpenFisca
-PORT=$OPENFISCA_PORT nohup ./start.sh mes-aides >> ../openfisca.log 2>> ../openfisca_error.log &
+{
+    forever stop openfisca
```

Comment:
Other solution: split [`start.sh`](https://github.com/sgmap/openfisca/blob/master/start.sh) functionality between creating the `current.ini` file and actually starting `paster`, and use `forever` here. You still need to keep `start.sh` in the other repo for easy usage, but it makes sense that in the current production context, we use another way to start the server.

> Using `forever` in `start.sh` is a no-go for me, as it adds complexity to the `openfisca-ops` module, which is supposed to be simple and is already quite complex as it handles workarounds. Even if it could make sense that it one day handles production constraints, I don't think `forever`, as a JS tool, makes sense as a dependency of a Python project.